requirements.txt

```diff
@@ -32,7 +32,7 @@ xlrd
 imageio>=2.6.1
 imageio-ffmpeg>=0.4.1
 traitlets
-pyvista>=0.32,!=0.35.2,!=0.38.0,!=0.38.1,!=0.38.2,!=0.38.3
+pyvista>=0.32,!=0.35.2,!=0.38.0,!=0.38.1,!=0.38.2,!=0.38.3,!=0.38.4
```

Comment:
this is getting ridiculous: 5 releases in a row are not working for us. Can we somehow upstream a test that uses MNE? cc @banesullivan

Reply:
https://github.com/pyvista/pyvista/pull/4036 https://github.com/pyvista/pyvistaqt/pull/317

Reply:
(the failure here ends up showing up as a doc build failure, though, so the integration test itself does not catch it)

Reply:
I just returned from vacation... will look into those issues to help resolve this.